src/lib/storage/disk_size.rb

```diff
@@ -204,7 +204,7 @@ def to_human_readable
         unit_index = 0
         size = @size_k.to_f
 
-        while size > 1024.0 && unit_index < UNITS.size - 1
+        while size >= 1024.0 && unit_index < UNITS.size - 1
```

Comment:
Is this the fix of a bug? If so, at some point (better sooner than later) we should add a test to avoid regression.